src/script/main/app.ts

```diff
@@ -606,7 +609,7 @@ class App {
       }
     }
 
-    await container.resolve(StorageService).init(userEntity.id);
+    await container.resolve(StorageService).init(this.core.storage);
```

Comment:
Later on, we can finally init the storage engine with the storage engine that the core has created